oak-run-elastic/src/main/java/org/apache/jackrabbit/oak/index/indexer/document/ElasticIndexer.java

```diff
@@ -126,6 +126,11 @@ public Set<String> getRelativeIndexedNodeNames() {
         return definition.getRelativeNodeNames();
     }
 
+    @Override
+    public String getIndexName() {
+        return definition.getIndexName();
```

Comment:
I think the usage of getIndexName() is fine. However, just FYI:

For on-prem and managed services, we support the same node name in multiple locations. For example it's possible to have 

* /oak:index/damAssetLucene
* /content/dam/oak:index/damAssetLucene

both in the same repository.

It is unlikely that this causes issues for this case here however (in my view).

Reply:
`definition.getIndexName()` returns the full path of the index, so there is no risk of collisions for indexes that are outside `/oak:index`.